### ISSUE TITLE
Focus on click instead of double click

### ISSUE THREAD

**Issue by markronquillo** [denisraslov/react-spreadsheet-grid#19]
Hi, 

May I ask if it is possible to set focus in the textbox inside a cell after the first click instead of double clicking it?

Thanks

**Comment by denisraslov** [comment]
Hi!
Right now it's not possible, but I think it could be useful if we add a new property to Grid for this workflow. Maybe I will do that in one of the next releases. You could do it as well, PR is welcome 🙂

**Comment by markronquillo** [comment]
Yeah, I'll work on it, since I need it. I'll try to make a PR this week. thanks man

**Comment by denisraslov** [comment]
Thank you!

What about an implementation, I think you have to change the `onCellClick` method of https://github.com/denisraslov/react-spreadsheet-grid/blob/master/src/grid/index.js. Now it only put `focusedCell` to the state, you have to put `activeCell` as well if we have a new property set for that.

```js
    onCellClick(x, y, row, columnId, e) {
        if (!find(this.props.disabledCells, { x, y })) {
            if (!e.skipCellClick && !isEqual(this.state.focusedCell, { x, y })) {
                this.setState({
                    focusedCell: e.target !== e.currentTarget ? { x, y } : null,
                    activeCell: { x, y }
                });
            }
        }

        if (this.props.onCellClick) {
            this.props.onCellClick(row, columnId);
        }

        this.skipGlobalClick = true;
    }
```

**Comment by markronquillo** [comment]
Added a PR here #20

**Comment by markronquillo** [comment]
What I did is to forcefully set the focusedCell, bypassing the `e.target !== e.currentTarget` condition using a newly defined prop. You can view it in the PR.

Tell me if you need me to fix or adjust anything. Thanks
